docs/ocis/development/testing.md

```diff
@@ -89,15 +89,12 @@ make -C tests/acceptance/docker clean
 
 ## Testing with test suite natively installed
 
-We are using the ownCloud 10 acceptance test suite against oCIS.
+we have 2 sets of tests:
+- `test-acceptance-core-api` set was transferred from [core](https://github.com/owncloud/core) repository 
+The suite name of all tests transferred from the core starts with "core"
 
-### Getting the tests
+- `test-acceptance-api` set was created for ocis. Mainly for testing spaces features
```

Comment:
As the tests for core have been named: `test-acceptance-core-api`, would it make sense to harmonize naming and rename the ocis tests to `test-acceptance-ocis-api`? This would make things imho much clearer and you never know what will come in future...

Reply:
We agreed to rename` test-acceptance-core-api` to `test-acceptance-from-core-api`